lnclient/lnd/lnd.go

```diff
@@ -313,34 +313,54 @@ func (svc *LNDService) LookupInvoice(ctx context.Context, paymentHash string) (t
 	return transaction, nil
 }
 
+func (svc *LNDService) getPaymentResult(stream routerrpc.Router_SendPaymentV2Client) (*lnrpc.Payment, error) {
+	for {
+		payment, err := stream.Recv()
+		if err != nil {
+			return nil, err
+		}
+
+		if payment.Status != lnrpc.Payment_IN_FLIGHT {
+			return payment, nil
+		}
+	}
+}
+
 func (svc *LNDService) SendPaymentSync(ctx context.Context, payReq string, amount *uint64) (*lnclient.PayInvoiceResponse, error) {
-	sendRequest := &lnrpc.SendRequest{PaymentRequest: payReq}
+	const MAX_PARTIAL_PAYMENTS = 21
+	const SEND_PAYMENT_TIMEOUT = 60
+	sendRequest := &routerrpc.SendPaymentRequest{
+		PaymentRequest: payReq,
+		MaxParts:       MAX_PARTIAL_PAYMENTS,
+		TimeoutSeconds: SEND_PAYMENT_TIMEOUT,
+		FeeLimitSat:    math.MaxInt64,
```

Comment:
This is set to max currently because leaving this would assume that we want 0-fee routes and would result in no route errors. This should be changed with the value from `CalculateFeeReserveMsat`

Reply:
hmm, I thought LND set a reasonable default, but anyway good to point this out, please make sure to update both places as soon as we merge the other PR

Reply:
Yeah there should've been a default for this :/

> please make sure to update both places as soon as we merge the other PR

Will do 🫡